lib/github.js

```diff
@@ -293,6 +293,35 @@ function parsePRBody(body) {
   );
 }
 
+/**
+ * @param {PullRequestOpt} pullRequest
+ * @param {string} label
+ * @returns {Promise<Boolean>}
+ */
+async function prHasLabel(pullRequest, label) {
+  const {number, repoName} = pullRequest;
+  const [owner, repo] = repoName.split('/');
+
+  return github
+    .graphql(
+      `query($owner: String!, $repo: String!, $number: Int!) {
+        repository(owner: $owner, name: $repo) {
+          pullRequest(number: $number) {
+            labels(first: 20) {
```

Comment:
There goes my plan to add all of my favorite pokemon as labels.

Reply:
`first: 1000`?